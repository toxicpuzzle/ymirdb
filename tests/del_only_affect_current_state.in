SET a 1 2 3
SET b 4 5 6
SET c 7 8 9
PUSH a b
PUSH b c
APPEND a c
SNAPSHOT
DEL a 
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
DEL a
LIST ENTRIES
ROLLBACK 1
LIST ENTRIES
DEL a
LIST ENTRIES
ROLLBACK 1
DROP 1
LIST ENTRIES
bye