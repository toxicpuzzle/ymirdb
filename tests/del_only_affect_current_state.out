> ok

> ok

> ok

> ok

> ok

> ok

> saved as snapshot 1

> ok

> c [7 8 9]
b [c 4 5 6]

> ok

> c [7 8 9]
b [c 4 5 6]
a [b 1 2 3 c]

> ok

> c [7 8 9]
b [c 4 5 6]

> ok

> c [7 8 9]
b [c 4 5 6]
a [b 1 2 3 c]

> ok

> c [7 8 9]
b [c 4 5 6]

> ok

> ok

> c [7 8 9]
b [c 4 5 6]
a [b 1 2 3 c]

> bye
